profiles/multisite_drupal_communities/modules/custom/multisite_drupal_access/multisite_drupal_access.info

```diff
@@ -1,7 +1,6 @@
 name = multisite drupal access
 description = module to customize the group node access
 package = multisite drupal toolbox
-core = 7.x
 
 ; Information added by drupal.org packaging script on 2011-11-24
 version = "7.x-1.0"
```

Comment:
This contains the packaging information from drupal.org for some bizarre reason. I did a search but this module never seems to have been published on drupal.org. Probably this info file was copy/pasted from a contributed module.

Let's add back the `core = 7.x` line, and remove the packaging information.